### PR TITLE
Add colors to the tui monitor view

### PR DESCRIPTION
**Pull request by wjhoward**
I've really enjoyed using Oha, thank you for your hard work. 

This PR adds color to the main monitor view, using the Bang Wong color blind friendly palette. This can be optionally disabled using the `--disable-color` flag.

I did consider adding terminal capable color checks, e.g. checking `$COLORTERM` / `$TERM` environment variables, although this doesn't work for all platforms (e.g. Windows Powershell). All modern terminals will support colors, and I've tested this on MacOs - Iterm2 and Windows 10 Powershell ( I don't have an Ubuntu Gnome terminal to hand ).

image

I did consider adding color ranges to the printed latency output, as per issue: https://github.com/hatoo/oha/issues/64 but I hesitated, as latency ranges can vary considerably per application.

The README etc would need to be updated, but wanted to get your thoughts on the change first.

### CODE REVIEW

**Comment by hatoo** [comment]
Great👍 
Please go ahead.
Don't forget to update `CHANGELOG.md`!

**Comment by wjhoward** [comment]
Great, thanks! Do you want me to update the `demo.gif` also?

**Comment by hatoo** [comment]
Yeah, if you update `demo.gif` also, it's very helpful.

**Comment by wjhoward** [comment]
> Yeah, if you update `demo.gif` also, it's very helpful.

Added a new `demo.gif`, the dimensions are lower than the original, but this seems largely due to the fact I tried to keep the amount of detail the same in the gif using a small terminal window. If I draw a bigger window (with higher dimensions) I get much more detail, e.g. many more bars. It looks pretty good in the README, e.g. https://github.com/wjhoward/oha/tree/add-tui-colors

**Comment by wjhoward** [comment]
Let me know if you want me to make any changes or add anything else before considering merging this? Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+# 0.5.5 (2022-09-12)
```
Comment (hatoo) [inline CHANGELOG.md:3]: Please remove this line.
I'll add this line when we release a new version

**Review by hatoo — CHANGES_REQUESTED** [review]

**Comment by hatoo** [comment]
Sorry, I thought I added a review yesterday but it was `pending` state.
I've added reviews now, could you see my reviews?

**Comment by wjhoward** [comment]
> Sorry, I thought I added a review yesterday but it was `pending` state. I've added reviews now, could you see my reviews?

No problem! I can see the review now, will make the required changes shortly. Thanks

**Review by hatoo — APPROVED** [review]

**Comment by hatoo** [comment]
Thanks! merged.

**Comment by Vanav** [comment]
I don't see colors in tui for some reason: 
image
Also notice that progress bar is invisible in "color" mode.

oha v0.5.6 (amd64 build from Releases page), Ubuntu 22 / Debian 11, TERM=xterm-256color.
Ternimal has no issues with colors, even oha shows some: 
image

**Pull request merged**